app/views/course/assessment/question/multiple_responses/_option_fields.html.slim

```diff
@@ -1,6 +1,10 @@
 = content_tag_for(:tr, f.object, class: 'nested-fields') do
+  td.reorder.handle
+    span = fa_icon 'reorder'.freeze
   td = f.input :correct, label: false
   td = f.input :option, label: false, input_html: { class: ['multiple-response-option', 'airmode'] }
-  td = f.input :explanation, label: false, placeholder: t('.explanation_hint'),
+  td
+    = f.input :explanation, label: false, placeholder: t('.explanation_hint'),
     input_html: { class: ['multiple-response-explanation', 'airmode'] }
   td = link_to_remove_association t('.remove'), f
+  td.weight = f.input :weight, as: :integer, label: false
```

Comment:
I think you can set this to be hidden by default, and not require the javascript to hide (ie. `$('.weight').hide();`), unless you are thinking of making it manually editable for users without javascript?